spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java

```diff
@@ -1015,7 +1025,11 @@ private void parseHostAndPort(String input) {
 			int portIndex = input.indexOf(':');
 			if (portIndex == -1) {
 				this.host = input;
-				this.port = DEFAULT_PORT;
+				if (this.isSecureConnection) {
+					this.port = DEFAULT_PORT_SECURE;
+				} else {
```

Comment:
@markpollack you likely already caught this but saw the ci failure and this should be 
```suggestion
				} 
				else {
```

Based upon the [checkstyle rules](https://ci.spring.io/builds/94734#L5d9fa795:3102).

```
[INFO] --- maven-checkstyle-plugin:3.0.0:check (checkstyle-validation) @ spring-boot-autoconfigure ---
[INFO] There is 1 error reported by Checkstyle 8.22 with src/checkstyle/checkstyle.xml ruleset.
[ERROR] src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java:[1030,33] (blocks) RightCurly: '}' at column 5 should be alone on a line.
```

Reply:
Actually I didn't notice, i'll look into it.